.Rbuildignore

```diff
@@ -5,9 +5,6 @@
 ^\./Example/example_multi\.R$
 ^\.travis\.yml$
 ^appveyor\.yml$
-^tmp-tests$
```

Comment:
These are actually useful, at least locally for me.
It is not a problem to have too many things in this file.